### PR TITLE
[CI:DOCS] Add release notes for Podman v2.1.0-RC1

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by rhatdan** [comment]
If we are shipping with buildah 1.16.1 (Which we should) we should make some noise about big improvements in the COPY/ADD Parsing on podman build, specifically around .dockerignore files.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,95 @@
 # Release Notes
 
+## 2.1.0
+### Features
+- A new command, `podman image mount`, has been added. This allows for an image to be mounted, read-only, to inspect its contents without creating a container from it ([#1433](https://github.com/containers/podman/issues/1433)).
+- The `podman save` and `podman load` commands can now create and load archives containing multiple images ([#2669](https://github.com/containers/podman/issues/2669)).
+- Rootless Podman now supports all `podman network` commands, and rootless containers can now be joined to networks.
+- The `podman run` and `podman create` commands now support a new mode for the `--cgroups` flag, `--cgroups=split`. Podman will create two cgroups under the cgroup it was launched in, one for the container and one for Conmon. This mode is useful for running Podman in a systemd unit, as it ensures that all processes are retained in systemd's cgroup hierarchy ([#6400](https://github.com/containers/podman/issues/6400)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:8]:
```suggestion
- The `podman run` and `podman create` commands now support a new mode for the `--cgroups` option, `--cgroups=split`. Podman will create two cgroups under the cgroup it was launched in, one for the container and one for Conmon. This mode is useful for running Podman in a systemd unit, as it ensures that all processes are retained in systemd's cgroup hierarchy ([#6400](https://github.com/containers/podman/issues/6400)).
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman save` and `podman load` commands can now create and load archives containing multiple images ([#2669](https://github.com/containers/podman/issues/2669)).
+- Rootless Podman now supports all `podman network` commands, and rootless containers can now be joined to networks.
+- The performance of `podman build` on `ADD` and `COPY` instructions has been greatly improved, especially when a `.dockerignore` is present.
+- The `podman run` and `podman create` commands now support a new mode for the `--cgroups` flag, `--cgroups=split`. Podman will create two cgroups under the cgroup it was launched in, one for the container and one for Conmon. This mode is useful for running Podman in a systemd unit, as it ensures that all processes are retained in systemd's cgroup hierarchy ([#6400](https://github.com/containers/podman/issues/6400)).
+- The `podman run` and `podman create` commands can now specify options to slirp4netns by using the `--network` flag as follows:  `--net slirp4netns:opt1,opt2`. This allows for, among other things, switching the port forwarder used by slirp4netns away from rootlessport.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:10]:
```suggestion
- The `podman run` and `podman create` commands can now specify options to slirp4netns by using the `--network` option as follows:  `--net slirp4netns:opt1,opt2`. This allows for, among other things, switching the port forwarder used by slirp4netns away from rootlessport.
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman ps` command now features a new flag, `--storage`, to show containers from Buildah, CRI-O and other applications.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:11]:
```suggestion
- The `podman ps` command now features a new option, `--storage`, to show containers from Buildah, CRI-O, and other applications.
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman run` and `podman create` commands now feature a `--sdnotify` flag to control the behavior of systemd's sdnotify with containers, enabling improved support for Podman in `Type=notify` units.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:12]:
```suggestion
- The `podman run` and `podman create` commands now feature a `--sdnotify` option to control the behavior of systemd's sdnotify with containers, enabling improved support for Podman in `Type=notify` units.
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman run` command now features a `--preserve-fds` flag to pass file descriptors from the host into the container ([#6458](https://github.com/containers/podman/issues/6458)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:13]:
```suggestion
- The `podman run` command now features a `--preserve-fds` option to pass file descriptors from the host into the container ([#6458](https://github.com/containers/podman/issues/6458)).
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman run` and `podman create` commands can now create overlay volume mounts, by adding the `:O` option to a bind mount (e.g. `-v /test:/test:O`). Overlay volume mounts will mount a directory into a container from the host and allow changes to it, but not write those changes back to the directory on the host.
+- The `podman play kube` command now supports the Socket HostPath type ([#7112](https://github.com/containers/podman/issues/7112)).
+- The `podman play kube` command now supports read-only mounts.
+- The `podman play kube` command now properly handles `HostAlias` entries.
+- The `podman generate kube` command now adds entries to `/etc/hosts` from `--host-add` generated YAML as `HostAlias` entries.
+- The `podman play kube` and `podman generate kube` commands now properly support `shareProcessNamespace` to share the PID namespace in pods.
+- The `podman volume ls` command now supports the `dangling` filter to identify volumes that are dangling (not attached to any container).
+- The `podman run` and `podman create` commands now feature a `--umask` flag to set the umask of the created container.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:21]:
```suggestion
- The `podman run` and `podman create` commands now feature a `--umask` option to set the umask of the created container.
```

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman create` and `podman run` commands now feature a `--tz` flag to set the timezone within the container ([#5128](https://github.com/containers/podman/issues/5128)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:22]:
```suggestion
- The `podman create` and `podman run` commands now feature a `--tz` option to set the timezone within the container ([#5128](https://github.com/containers/podman/issues/5128)).
```

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -1,5 +1,96 @@
...
+- Environment variables for Podman can now be added in the `containers.conf` configuration file.
+- The `--mount` flag to `podman run` and `podman create` now supports a new mount type, `type=devpts`, to add a `devpts` mount to the container. This is useful for containers that want to mount `/dev/` from the host into the container, but still create a terminal.
+- The `--security-opt` flag to `podman run` and `podman create` now supports a new option, `proc-opts`, to specify options for the container's `/proc` filesystem.
+- Podman with the `crun` OCI runtime now supports a new flag to `podman run` and `podman create`, `--cgroup-conf`, which allows for advanced configuration of cgroups on cgroups v2 systems.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:26]:
```suggestion
- Podman with the `crun` OCI runtime now supports a new option to `podman run` and `podman create`, `--cgroup-conf`, which allows for advanced configuration of cgroups on cgroups v2 systems.
```

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -1,5 +1,96 @@
...
+- The `podman create` and `podman run` commands now support a `--override-variant` flag, to override the architecture variant of the image that will be pulled and ran.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:27]:
```suggestion
- The `podman create` and `podman run` commands now support a `--override-variant` option, to override the architecture variant of the image that will be pulled and ran.
```

File: RELEASE_NOTES.md [hunk 10]
```diff
@@ -1,5 +1,96 @@
...
+- A new global option has been added to Podman, `--runtime-flags`, which allows for setting flags on calls to the OCI runtime.
+- The `podman manifest add` command now supports the `--cert-dir`, `--auth-file`, `--creds`, and `--tls-verify` flags.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:29]:
```suggestion
- The `podman manifest add` command now supports the `--cert-dir`, `--auth-file`, `--creds`, and `--tls-verify` options.
```

File: RELEASE_NOTES.md [hunk 11]
```diff
@@ -1,5 +1,96 @@
...
+- Fixed a bug where rootless Podman would ignore errors from `newuidmap` and `newgidmap`, even if `/etc/subuid` and `/etc/subgid` contained valid mappings for the user running Podman.
+- Fixed a bug where the `podman commit` command did not properly handle single-character image names ([#7114](https://github.com/containers/podman/issues/7114)).
+- Fixed a bug where the output of `podman ps --format=json` did not include a `Status` field ([#6980](https://github.com/containers/podman/issues/6980)).
+- Fixed a bug where input to the `--log-level` option was no longer case-insensitive.
+- Fixed a bug where `podman images` could segfault when an image pull was aborted while incomplete, leaving an image without a manifest ([#7444](https://github.com/containers/podman/issues/7444)).
+- Fixed a bug where rootless Podman would try to create the `~/.config` directory when it did not exist, despite not placing any configuration files inside the directory.
+- Fixed a bug where the output of `podman system df` was incosistent based on whether the `-v` option was specified ([#7405](https://github.com/containers/podman/issues/7405)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:49]:
```suggestion
- Fixed a bug where the output of `podman system df` was inconsistent based on whether the `-v` option was specified ([#7405](https://github.com/containers/podman/issues/7405)).
```

**Comment by mheon** [comment]
Comments addressed, anyone want to lgtm/hold this?

File: RELEASE_NOTES.md [hunk 12]
```diff
@@ -1,5 +1,96 @@
...
+- Fixed a bug where the Compat Create endpoint for containers did not properly handle Entrypoint and Command from images.
+- Fixed a bug where name history information was not properly added in the Libpod Image List endpoint.
+- Fixed a bug where the Libpod image search endpoint improperly populated the Description field of responses.
+- Added a `noTrunc` option to the Libpod image search endpoint.
+- Fixed a bug where the Pod List API would return null, instead of an empty array, when no pods were present ([#7392](https://github.com/containers/podman/issues/7392)).
+- Fixed a bug where endpoints that hijacked would do perform the hijack too early, before being ready to send and receive data ([#7195](https://github.com/containers/podman/issues/7195)).
+- Fixed a bug where Pod endpoints that can operate on multiple containers at once (e.g. Kill, Pause, Unpause, Stop) would not forward errors from individual containers that failed.
+- The Compat List endpoint for networks now supports filtering results ([#7462](https://github.com/containers/podman/issues/7462)).
```
Comment (Luap99) [inline RELEASE_NOTES.md:86]: Compat Network List

File: RELEASE_NOTES.md [hunk 13]
```diff
@@ -1,5 +1,96 @@
...
+- Fixed a bug where `podman --user` with a numeric user and `podman run --userns=keepid` could create users that could not be logged into ([#7499](https://github.com/containers/podman/issues/7499)).
+- Fixed a bug where trying to join another container's user namespace with `--userns container:$ID` would fail ([#7547](https://github.com/containers/podman/issues/7547)).
+- Fixed a bug where the `podman play kube` command would trim underscores from container names ([#7020](https://github.com/containers/podman/issues/7020)).
+- Fixed a bug where the `podman attach` command would not show output when attaching to a container with a terminal ([#6523](https://github.com/containers/podman/issues/6253)).
+- Fixed a bug where the `podman system df` command could be extremely slow when large quantities of images were present ([#7406](https://github.com/containers/podman/issues/7406)).
+
+### API
+- Docker-compatible Volume Endpoints (Create, Inspect, List, Remove, Prune) are now available!
+- Added an endpoint for generating Systemd YAML for containers.
```
Comment (Luap99) [inline RELEASE_NOTES.md:72]: systemd units?!

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: RELEASE_NOTES.md (hunk 12 above)
Comment (Luap99) [reply RELEASE_NOTES.md:86]: Nevermind, my bad

File: RELEASE_NOTES.md (hunk 13 above)
Comment (mheon) [reply RELEASE_NOTES.md:72]: Oh. Oops. Good catch.

**Comment by edsantiago** [comment]
/lgtm

Release the hold when desired. Thanks @mheon.

**Pull request merged**